vignettes/intro_opencage.Rmd.src

```diff
@@ -0,0 +1,188 @@
+---
+title: "Introduction to opencage"
+subtitle: "Forward and Reverse Geocoding"
+author: "Daniel Possenriede, Jesse Sadler, Maëlle Salmon"
+date: "`r Sys.Date()`"
+description: >
+  "Get started with the opencage R package to geocode with the OpenCage API, either from placename to longitude and latitude (forward geocoding) or from longitude and latitude to placename (reverse geocoding)."
+output: 
+  rmarkdown::html_vignette:
+    df_print: kable
+vignette: >
+  %\VignetteIndexEntry{Introduction to opencage}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, echo = FALSE, warning=FALSE, message=FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+
+library("opencage")
+
+```
+
+Geocoding is the process of converting placenames or addresses into geographic coordinates, like latitude and longitude, or vice versa. 
+With {opencage} you can geocode using the OpenCage API, either from placename to longitude and latitude (forward geocoding) or from longitude and latitude to placename (reverse geocoding). 
+
+## Setup
+
+Before you can use the {opencage} package and query the OpenCage API you need to set up authentication, possibly want to set a rate limit (if you have a paid [OpenCage plan](https://opencagedata.com/pricing)) and maybe want to prevent OpenCage from storing the content of your queries, i.e. you need to setup {opencage}. 
+
+### Authentication
+
+To use the package and authenticate yourself against the OpenCage API, you will need to register at [opencagedata.com/users/sign_up](https://opencagedata.com/users/sign_up) to get an API key. 
+The "Free Trial" plan provides up to 2,500 API requests a day. 
```

Comment:
maybe we should not add numbers here since plans could change

Reply:
I find that info (no. of requests per day) rather important. OpenCage is pretty stable in that regard, or at least I don't think that is has changed (for any plan?) in the past. 
A lot of info that we have in the vignettes can change any day, so I don't think that the number here is special in that regard.